### ISSUE TITLE
ubuntu trusty release does not have a release file

### ISSUE THREAD

**Issue by CodyFraker** [bhemar/raspberry-metrics#5]
Trying to install this on my rPi model 4 running Raspbian 10, receiving this error when trying to run the ansible playbook.

`fatal: [192.168.1.171]: FAILED! => {"changed": false, "cmd": "apt-get update", "msg": "E: The repository 'http://ppa.launchpad.net/ansible/ansible/unbuntu trusty Release' does not have a Release file.", "rc": 100, "stderr": "E: The repository 'http://ppa.launchpad.net/ansible/ansible/unbuntu trusty Release' does not have a Release file.\n", "stderr_lines": ["E: The repository 'http://ppa.launchpad.net/ansible/ansible/unbuntu trusty Release' does not have a Release file."], "stdout": "Hit:1 http://archive.raspberrypi.org/debian buster InRelease\nGet:2 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]\nIgn:3 http://ppa.launchpad.net/ansible/ansible/unbuntu trusty InRelease\nErr:4 http://ppa.launchpad.net/ansible/ansible/unbuntu trusty Release\n  404  Not Found [IP: 91.189.95.83 80]\nReading package lists...\n", "stdout_lines": ["Hit:1 http://archive.raspberrypi.org/debian buster InRelease", "Get:2 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]", "Ign:3 http://ppa.launchpad.net/ansible/ansible/unbuntu trusty InRelease", "Err:4 http://ppa.launchpad.net/ansible/ansible/unbuntu trusty Release", "  404  Not Found [IP: 91.189.95.83 80]", "Reading package lists..."]}`

**Comment by bhemar** [comment]
Hello,

I have tried this Ansible playbook on the latest Rasbian OS (Rasbian Buster lite) on my Rasbperry (Raspberry PI 2 model B). Everything was installed and configured successfully.

Can you run apt-get update & upgrade manually?

`sudo apt-get update`
`sudo apt-get upgrade`

**Comment by CodyFraker** [comment]
Sure thing. Here is the output when running the update command.

`sudo apt-get update

Ign:1 http://ppa.launchpad.net/ansible/ansible/unbuntu trusty InRelease
Hit:2 http://archive.raspberrypi.org/debian buster InRelease
Get:3 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]
Err:4 http://ppa.launchpad.net/ansible/ansible/unbuntu trusty Release
  404  Not Found [IP: 91.189.95.83 80]
Reading package lists... Done
E: The repository 'http://ppa.launchpad.net/ansible/ansible/unbuntu trusty Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.`

also says apt-secure isn't a valid command


I didn't bother running the next command since it through an error on the update. 

Awesome work by the way - I can't wait to get this operational 👍 
Thanks for all your advice and help



edit: I just noticed I have a typo somewhere in my commands. Let me check on that and ill post another comment with a followup.

**Comment by CodyFraker** [comment]
I'm an idiot and just had a typo in my sources.

Fixed this issue which was my fault and not your code. Went into /etc/apt/sources.list and did
sudo vim /etc/apt/sources.list
fixed the typo, ran those commands. its working again. Going to re-run through the process of setting up this stuff since it wasn't working before. I had to go install grafana myself. Will open another issue if things go wrong.
